recipes/napari-subboxer/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "napari-subboxer" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-subboxer-{{ version }}.tar.gz
+  sha256: 381a9e95c505e9d2240428287876ac45bb656566071b48f51e2d1f13fa2c2088
+
+build:
+  entry_points:
+    - napari-subboxer = napari_subboxer.cli:cli
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools-scm
+  run:
+    - einops
+    - eulerangles
+    - mrcfile
+    - napari ==0.4.12
```

Comment:
This one is a bit too strict. Are any other versions known to work with this plugin @alisterburt ?

Reply:
Sorry about this, I'm using an experimental API in napari for this plugin which has been set to change 'soon' (napari/napari#3759) for a while now

I just tested and the plugin currently runs with napari 0.4.12 but is broken in a few different places on 0.4.13 - I can try to sort it out soon but no guarantees on timescale

Reply:
No probs!